src/test/regress/expected/auto_grant_all_to_regularuser.out

```diff
@@ -0,0 +1,5 @@
+ALTER DEFAULT PRIVILEGES GRANT ALL ON SCHEMAS TO regularuser; -- does not work with citus
```

Comment:
can we do this conditionally, only when `session_role` is `superuser`?

Because, most of the time, the owner of the objects is already the regular user